server/fishtest/templates/tests_run.mak

```diff
@@ -335,10 +335,10 @@
   });
 
```

Comment:
Maybe add a comment here:
```
## In the future, this should be changed to normalized Elo,
## and converted within fishtest to logistic Elo whenever needed.
## See also https://github.com/glinscott/fishtest/issues/865#issuecomment-808808220
```

Reply:
Not so sure how to enter a comment in a .mak file, in fact :-)

Are they html files? javascript files? python files?

Reply:
@ppigazzini @tomtor What is the best way to put comment in a .mak file?

Reply:
Mako template inline comment : `##`
https://stackoverflow.com/questions/20906838/markdown-conflict-in-mako

Reply:
@vondele wrote the comment the right way :)

Reply:
thanks! :-)

Reply:
yes, I actually had a patch prepared that matched exactly yours except for the comments :-)